bounceview/src/main/java/com/github/hariprasanths/bounceview/CustomDialog.java

```diff
@@ -0,0 +1,243 @@
+                .parse(ResourceTable.Layout_title, null, false);
+        baseTitleLayout.setLayoutConfig(layoutConfig);
+        baseContentLayout = LayoutScatter.getInstance(context)
+                .parse(ResourceTable.Layout_content, null, false);
+        baseContentLayout.setLayoutConfig(layoutConfig);
+        baseButtonLayout = LayoutScatter.getInstance(context)
+                .parse(ResourceTable.Layout_buttons, null, false);
+        baseImageButtonLayout = LayoutScatter.getInstance(context)
+                .parse(ResourceTable.Layout_image_buttons, null, false);
+        baseButtonLayout.setLayoutConfig(layoutConfig);
+    }
+
+    public CommonDialog setComponentContainer(Component component) {
+        return super.setContentCustomComponent(component);
+    }
+
+    public Component getComponentContainer() {
+        return super.getContentCustomComponent();
+    }
+
+    @Override
+    public CommonDialog setTitleCustomComponent(DirectionalLayout component) {
+        mTitleComponent = component;
+        return this;
+    }
+
+    public Component getTitleCustomComponent() {
+        return mTitleComponent;
+    }
+
+    @Override
+    public CommonDialog setContentCustomComponent(Component component) {
+        mContentComponent = component;
+        return this;
+    }
+
+    @Override
+    public Component getContentCustomComponent() {
+        return mContentComponent;
+    }
+
+    private CommonDialog setButtonCustomComponent(Component component) {
+        mButtonComponent = component;
+        return this;
+    }
+
+    @Override
+    public CommonDialog setTitleText(String text) {
+        Text titleText = (Text) baseTitleLayout.findComponentById(ResourceTable.Id_titleText);
+        titleText.setText(text);
+        titleText.setVisibility(0);
+        this.setTitleCustomComponent((DirectionalLayout) baseTitleLayout);
+        return this;
+    }
+
+    @Override
+    public CommonDialog setTitleSubText(String text) {
+        Text titleText = (Text) baseTitleLayout.findComponentById(ResourceTable.Id_subTitleText);
+        titleText.setText(text);
+        titleText.setVisibility(Component.VISIBLE);
+        this.setTitleCustomComponent((DirectionalLayout) baseTitleLayout);
+        return this;
+    }
+
+    @Override
+    public CommonDialog setContentText(String text) {
+        Text contentText = (Text) baseContentLayout.findComponentById(ResourceTable.Id_contentText);
+        contentText.setText(text);
+        contentText.setVisibility(0);
+        this.setContentCustomComponent(baseContentLayout);
+        return this;
+    }
+
+    @Override
+    public CommonDialog setContentImage(int resId) {
+        Image image = (Image) baseContentLayout.findComponentById(ResourceTable.Id_imgContent);
+        image.setPixelMap(resId);
+        image.setVisibility(0);
+        this.setContentCustomComponent(baseContentLayout);
+        return this;
+    }
+
+    /**
+     * Does some thing in old style.
```

Comment:
Make the java doc as "This method is deprecated. Use setCommonButton()"

Reply:
done